src/views/Contact.vue

```diff
@@ -1,9 +1,9 @@
 <template>
```

Comment:
Okay, the skew looks good now but the heading is too small .

Also, can you smaller the icons in ContactItem? It breaks the words in the text in there.

![image](https://user-images.githubusercontent.com/27869979/98373272-4b1aad00-203f-11eb-9fca-141be492e86c.png)

![image](https://user-images.githubusercontent.com/27869979/98373287-5077f780-203f-11eb-8e1f-fce4b877138e.png)